apisix/core/config_etcd.lua

```diff
@@ -545,7 +558,34 @@ local function _automatic_fetch(premature, self)
 
             local ok, err = sync_data(self)
             if err then
-                if err ~= "timeout" and err ~= "Key not found"
+                if string.find(err, "connection refused")
+                    or string.find(err, "Service Unavailable") then
+                    log.warn(err, ", ", tostring(self))
```

Comment:
Is this for debugging purpose?

Reply:
Mainly to fulfill the needs of config-etcd test, and I think it's also good to inform users that etcd can not be connected
https://github.com/apache/apisix/blob/86e168e711713531eeafdd0a75682b150e0d0109/t/core/config_etcd.t#L28-L49

Reply:
Then the error message should be more semantic.

Reply:
This part got removed